changelog.d/5230.misc

```diff
@@ -0,0 +1 @@
+Remove urllib3 pin as requests 2.22.0 as been released supporting urllib3 1.25.2
```

Comment:
```suggestion
Remove urllib3 pin as requests 2.22.0 as been released supporting urllib3 1.25.2
```

Pedantic, I know :)

Reply:
I actually thought about this for a second, didn't know what to do and decided against adding the dot. :D

Buuut there is actually a type in there s/as been/has been/.

Reply:
Oh yeah huh.

Jokes on you though, we require the dot >:D https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#changelog

Reply:
now fixed.

Reply:
You mean I should read more than the first paragraph of contribution guideline docs? :thinking: ;-)